docs/data/base/components/menu/menu.md

```diff
@@ -17,61 +18,59 @@ import MenuUnstyled from '@mui/base/MenuUnstyled';
 import MenuItemUnstyled from '@mui/base/MenuItemUnstyled';
 ```
 
-The MenuUnstyled components can be used to create custom menus.
-It renders a list of items in a popup and allows mouse and keyboard navigation through them.
+You can use the `MenuUnstyled` component to create custom menus.
+It renders a list of items in a popup that users can navigate through with a mouse or keyboard.
 
-When not customized, the MenuItem renders a plain `ul` element.
+When not customized, the `MenuUnstyled` component renders a plain `<ul>` element.
 
 ### Basic usage
 
 {{"demo": "MenuSimple.js"}}
 
 ### Wrapping MenuItems
 
-MenuItemUnstyled components don't have to be direct descendants of the MenuUnstyled.
-Developers can wrap them in arbitrary components to achieve desired appearance.
+`MenuItemUnstyled` components don't have to be direct chilrden of a `MenuUnstyled` component. You can wrap them in any component needed to achieve the desired appearance.
 
-Additionally, MenuUnstyled may contain non-interactive children (such as help text).
+In addition to `MenuItemUnstyled` components, the `MenuUnstyled` component can also contain non-interactive children, such as help text.
 
 {{"demo": "WrappedMenuItems.js"}}
 
 ### Customization
 
 #### Slots
 
-The MenuUnstyled has two slots:
+`MenuUnstyled` has two slots:
 
-- Root - represents the popup container the menu is placed in.
+- Root - represents the popup container the menu is placed in; set to `PopperUnstyled` by default
```

Comment:
```suggestion
- Root - represents the popup container the menu is placed in; set to `PopperUnstyled` by default.
```

Otherwise, looks like this:

![image](https://user-images.githubusercontent.com/4512430/161737756-8bf7d7cd-8c4b-43d2-955e-dcc7a0a04802.png)